### PR TITLE
Reduce requests to proxy by one

### PR DESCRIPTION
**Pull request by zisra**
This pull request replaces the [external_ids request](https://developer.themoviedb.org/reference/movie-external-ids) with an [append to response parameter](https://developer.themoviedb.org/docs/append-to-response)

I feel like this is pretty important because it reduced the amount of proxy bandwidth used + increases speed.

### CODE REVIEW

**Comment by castdrian** [comment]
thonk, feature freeze + tmdb requests don't go through the proxy

**Comment by zisra** [comment]
Oh, but I guess one request less is always better and keeps users waiting less (300ms)

**Comment by castdrian** [comment]
true, still quite funny though

**Comment by zisra** [comment]
If I had the time I would probably do a crazy calculation to determine how much time all users would save in a month

File: public/config.js [hunk 1]
```diff
@@ -1,6 +1,6 @@
 window.__CONFIG__ = {
   // url must NOT end with a slash
-  VITE_CORS_PROXY_URL: "",
+  VITE_CORS_PROXY_URL: "https://rough.isra.workers.dev",
```
Comment (castdrian) [inline public/config.js]: leaked, should change the subdomain, and use the env file while deving

File: src/backend/metadata/tmdb.ts [hunk 2]
```diff
@@ -155,10 +152,10 @@ export function getMediaDetails<
   TReturn = MediaDetailReturn<T>
 >(id: string, type: T): Promise<TReturn> {
   if (type === "movie") {
-    return get<TReturn>(`/movie/${id}`);
+    return get<TReturn>(`/movie/${id}?append_to_response=external_ids`);
   }
   if (type === "show") {
-    return get<TReturn>(`/tv/${id}`);
+    return get<TReturn>(`/tv/${id}?append_to_response=external_ids`);
```
Comment (castdrian) [inline src/backend/metadata/tmdb.ts:158]:
```suggestion
    return get<TReturn>(`/tv/${id}`, { append_to_response: "external_ids" });
```

File: src/backend/metadata/types/tmdb.ts [hunk 3]
```diff
@@ -169,6 +172,9 @@ export interface TMDBMovieData {
   video: boolean;
   vote_average: number;
   vote_count: number;
+  external_ids: {
+    imdb_id: string;
```
Comment (castdrian) [inline src/backend/metadata/types/tmdb.ts:176]:
```suggestion
    imdb_id: string | null;
```

**Review by castdrian — CHANGES_REQUESTED** [review]

**Review by castdrian — APPROVED** [review]

File: src/backend/metadata/types/tmdb.ts (hunk 3 above)
Comment (zisra) [reply src/backend/metadata/types/tmdb.ts]: Why? Causes linter errors too

Comment (castdrian) [reply src/backend/metadata/types/tmdb.ts]: imdb id isn't guaranteed for shows, just fix the lint errors I guess

**Comment by mrjvs** [comment]
Thanks for the PR, but it's not a planned issue, nor is it a critical bug fix.
Because of the feature freeze for v4, it wont be merged.

See #348 for more information, ive added some clarifying details.

**Pull request closed without merging**